### ISSUE TITLE
Switch postcss-advanced-variables back to main package

### ISSUE THREAD

**Issue by kmonahan** [forumone/nextjs-project#101]
`postcss-advanced-variables` has had some new releases, so we should test out https://www.npmjs.com/package/postcss-advanced-variables and see whether we can switch off of my forked package back to the main one.

**Comment by SirJohn96** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [NEXTJS-101](https://trello.com/c/LyVyY03l/835-nextjs-101)

**Comment by SirJohn96** [comment]
@kmonahan Things seem to be fine from what I see at the moment. What were some of the issues encountered that led to the fork?

**Comment by kmonahan** [comment]
@SirJohn96 Weirdly, I got an email notification with your comment but I don't see it here. If your question still stands, the issuses were in using CSS variables in `@mixin` and `@include` rules. Looking back at the [commit where we switched to the fork](https://github.com/forumone/nextjs-project/pull/14/commits/18a44fc245013cb775bb4804da58daf2c9e0616e), one place where this came up was in the `set-link-vars` mixin.
